app/routes/provider.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+const providers = config.Reviews.providers.slice(1);
+const providerIds = providers.map(p => p.id);
+
+/**
+ * @module ember-osf-reviews
+ * @submodule routes
+ */
+
+/**
+ * @class Provider Route Handler
+ */
+export default Ember.Route.extend({
+    theme: Ember.inject.service(),
+    params: Ember.inject.service('pager'),
+
+    // Todo: Replace the use of hardcoded preprint provider list with an API request.
+
+    beforeModel(transition) {
+        const {slug = ''} = transition.params.provider;
+        const slugLower = slug.toLowerCase();
+        this.set('theme.id', slugLower);
+        if (providerIds.includes(slugLower)) {
+            if (slugLower !== slug) {
+                const {pathname} = window.location;
+                const pathRegex = new RegExp(`^/preprints/${slug}`);
+                window.location.pathname = pathname.replace(pathRegex, `/preprints/${slugLower}`);
+            }
+        } else {
+            this.replaceWith('page-not-found');
+        }
+    },
+    model(params) {
+        params["filter[provider]"] = this.get('theme.id');
+        delete params["page"];
+        return Ember.RSVP.hash({
```

Comment:
The model for this route should be just the provider. This logic would belong in the [index route](https://guides.emberjs.com/v2.14.0/routing/defining-your-routes/#toc_index-routes)